### ISSUE TITLE
strun --version gives stpipe version, not jwst version

### ISSUE THREAD

**Issue by jdavies-st** [spacetelescope/jwst#7470]
The following happens:

```
$ strun calwebb_detector1 --version
0.4.5
```

But this is not useful for the user, as I'm getting the version of `stpipe`, not the calibration pipeline version.  Much more useful would be telling me which version of `jwst.pipeline.calwebb_detector1` I'm using, i.e. the version of the `jwst` package.

```
$ strun calwebb_detector1 --version
1.9.4
```

As a user, I will almost never care about the version of `stpipe`, but I will very much care about the version of the pipeline.  Any general way to do this?

**Comment by drlaw1558** [comment]
+1; this would be great.  Same thing with calls to specific packages (e.g., strun cube_build).  Maybe some way to resolve the package called and return the version of that instead?

**Comment by nden** [comment]
@stscijgbot-jp

**Comment by jdavies-st** [comment]
I do note here that `stpipe` gives the `jwst` version

```
$ stpipe --version
stpipe: 0.5.0
jwst: 1.11.3
```

Perhaps `strun` can as well.

**Comment by stscijgbot-jp** [comment]
_This issue is tracked on JIRA as [JP-3326](https://jira.stsci.edu/browse/JP-3326)._

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-3326?focusedCommentId=660108#comment-660108):_

Fixed by spacetelescope/stpipe#98
